examples/src/main/java/com/cloudera/nav/plugin/client/examples/updatedResults/GetAllUpdatedResults.java

```diff
@@ -0,0 +1,32 @@
+package com.cloudera.nav.plugin.client.examples.updatedResults;
```

Comment:
make sure the apache license appears in every java file. This can be set in your IDE. I'll show you how